src/platforms/common/blackpill-f4/README.md

```diff
@@ -67,12 +78,24 @@ make PROBE_HOST=blackpill-f4x1cx SHIELD=1
   - Release BOOT0
 - Upload the firmware:
 ```
-./dfu-util -a 0 --dfuse-address 0x08000000:leave -R -D blackmagic.bin
+./dfu-util -d 0483:df11 --alt 0 -s 0x08004000:leave -D build_xxxxxx/blackmagic_xxxxxx_bootloader.bin
```

Comment:
Should be firmware.bin

Reply:
The format shown is the "standard" for naming binaries in the project repository.